layersvt/device_simulation.cpp

```diff
@@ -1482,7 +1482,7 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateInstance(const VkInstanceCreateInfo *pCreat
         dt->GetPhysicalDeviceProperties(physical_device, &pdd.physical_device_properties_);
 
         // Initialize PDD members to the actual Vulkan implementation's defaults.
-        if (get_physical_device_properties2_active || VK_VERSION_MINOR(requested_version) > 0) {
+        if (PhysicalDeviceData::HasExtension(physical_device, VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME) && (get_physical_device_properties2_active || VK_VERSION_MINOR(requested_version) > 0)) {
```

Comment:
I think this would better be placed as a guard around lines 1466-1471, where `get_physical_device_properties2_active` is set.

Reply:
Though it's strange to me that the extension could be enabled while it is not present... ?